internal/controller/storageclient_controller.go

```diff
@@ -247,7 +247,7 @@ func (r *StorageClientReconciler) reconcilePhases() (ctrl.Result, error) {
 			}
 			secret := &corev1.Secret{}
 			secret.Name = eResource.Name
-			secret.Namespace = r.storageClient.Namespace
+			secret.Namespace = r.OperatorNamespace
```

Comment:
w/ a simple search I could see this in more than 1 place, pls check and fix https://github.com/search?q=repo%3Ared-hat-storage%2Focs-client-operator%20r.storageClient.Namespace&type=code

Reply:
Updated accordingly